server/package.json

```diff
@@ -25,7 +25,7 @@
     "@azure/storage-blob": "10.5.0",
     "@nestjs/azure-database": "0.0.1-beta.0",
     "@nestjs/azure-func-http": "0.2.0",
-    "@nestjs/azure-storage": "1.0.2",
+    "@nestjs/azure-storage": "^1.0.2",
```

Comment:
we need to freeze the package version for now in case of any breaking change. So for the moment, just specify the exact version you need.